src/blocktree.rs

```diff
@@ -1261,7 +1261,7 @@ pub fn create_new_ledger(ledger_path: &str, genesis_block: &GenesisBlock) -> Res
     Ok((1, entries[0].id))
 }
 
-pub fn genesis<'a, I>(ledger_path: &str, keypair: &Keypair, entries: I) -> Result<()>
+pub fn genesis<'a, I>(ledger_path: &str, _keypair: &Keypair, entries: I) -> Result<()>
```

Comment:
Deleting unused keypair parameters often has widespread awesome consequences.

Reply:
Yeah, usually but here it should be safe since the keypair was only used for setting the "id" of the blobs in `index_blobs`. I've left it in though since at some point we'll be signing blobs. I can get rid of it altogether though. It's only used in some test functions right now.